src/stuf/tests/test_migrations.py

```diff
@@ -0,0 +1,172 @@
+import os
+
+from django.conf import settings
+from django.core.files import File
+
+from openforms.utils.tests.test_migrations import TestMigrations
+
+from ..constants import SOAPVersion
+
+TEST_CERTIFICATES = os.path.join(
+    settings.BASE_DIR,
+    "src",
+    "stuf",
+    "stuf_zds",
+    "tests",
+    "data",
+)
```

Comment:
if you use `pathlib.Path`, you get usually a bit more readable code and it's the modern way to deal with filesystem paths in Python.